### PR TITLE
fix: add more descriptive error message to error thrown when attempt to prune FBC

### PR DESCRIPTION
**Pull request by jbuchananr**
**Description of the change:**
Provides a more descriptive error when `opm index prune` is called on a file based catalog letting users know that it only supports sqlite-based catalogs. 

**Motivation for the change:**
Currently, an unclear error message is thrown while calling `opm index prune` on a file based catalog. 
E.g `"Error: index image xxxx missing label operators.operatorframework.io.index.database.v1"`

Fixes [Bug2039135](https://bugzilla.redhat.com/show_bug.cgi?id=2039135#)

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: pkg/lib/indexer/indexer.go [hunk 1]
```diff
@@ -363,6 +363,9 @@ func (i ImageIndexer) getDatabaseFile(workingDir, fromIndex, caFile string, skip
 
 	dbLocation, ok := labels[containertools.DbLocationLabel]
 	if !ok {
+		if _, ok := labels[containertools.ConfigsLocationLabel]; ok {
```
Comment (perdasilva) [inline pkg/lib/indexer/indexer.go:366]: I think it might be worth adding a regression test

Comment (jbuchananr) [reply pkg/lib/indexer/indexer.go:366]: Working on it! +1

**Pull request merged**